linux_kernel_modules/cypwifi/scripts/cywifi.sh

```diff
@@ -28,6 +29,11 @@ fi
 uname -a | grep mdm9x28
 if [ $? -eq 0 ] ; then
 	CF3="mdm9x28"
+	# DV3 uses GPIO33 and DV4 uses wl_reg_on on the expander
+	BOARD_REV=`head -n 2 /sys/bus/i2c/devices/4-0050/eeprom | tail -1 | sed 's/Rev: //'`
+	if [ ${BOARD_REV} == "DV3" ] ; then
```

Comment:
As discussed with Dave, this should work for all versions of programmed DV3 EEPROM content.
```
if head -n 2 /sys/bus/i2c/devices/4-0050/eeprom | grep 'DV3'
```